src/extension.ts

```diff
@@ -382,6 +399,21 @@ export function defineGlobals(tempPath: string | undefined) {
     DS_DIR = path.join(BRIGHTTEMPFOLDER, "_D_");
 }
 
+/**
+ * Imports the neccesary security modules
+ */
+export function defineSecurity(moduleName) {
+    try {
+        return require(`${vscode.env.appRoot}/node_modules.asar/${moduleName}`);
+        // tslint:disable-next-line: no-empty
+      } catch (err) {}
+    try {
+        return require(`${vscode.env.appRoot}/node_modules/${moduleName}`);
+      // tslint:disable-next-line: no-empty
+      } catch (err) {}
+    return undefined;
+}
```

Comment:
If possible, I will convert this to a getter as well (to provide a more natural way of accessing/defining keytar.

I see where [this](https://code.visualstudio.com/api/advanced-topics/remote-extensions#persisting-secrets) was pulled from. However I do not believe we need to check for `node_modules.asar` since we are not using the asar node_module to package the extension.

That said, we may need to revisit this and confirm that this will work with webpack

Reply:
I suppose I consider a getter should return something. The only thing this returns is undefined hence I didn't use get. I will remove asar because we have the code documented in your extract if we need to put it back.
Does getSecurityModules sound correct?.